### PR TITLE
Add links to chat rooms, google groups, issue trackers, etc.

### LINKED ISSUE #1146: Add links to all our various chat rooms
it's rather painful to keep track of all of the ways in which the conda-forge community stays in touch. It'd be nice to enumerate them somewhere in our docs. The ones that I know of:

* conda-forge-core google group
* gitter.im for:
  * [core](https://gitter.im/conda-forge/core)
  * [public](https://gitter.im/conda-forge/conda-forge.github.io)
  * [power pc](https://gitter.im/conda-forge-ppc64le/Lobby)
* keybase

conda-tools stuff, kinda related
* conda google group
* conda slack org

What am I missing?

### PR DESCRIPTION
**Pull request by ericdill**
Closes #1146 

Two questions:
1. Did I miss anything?
2. Is putting the core stuff in here a bad idea?

### CODE REVIEW

File: src/orga/getting-in-touch.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+================
+
+There are many ways to get in touch with the conda-forge community, primarily through issue trackers, mailing lists and real-time chat.
+
+The main issue trackers that you will interact with are
+* [staged-recipes](https://github.com/conda-forge/staged-recipes/issues). You'll use staged-recipes to create a new conda package on conda-forge
+* [our main docs repo](https://github.com/conda-forge/conda-forge.github.io/issues). You'll use this repo as a catch-all for issues where you're not sure where else to put them
+* [our enhancement proposals repo](https://github.com/conda-forge/cfep/issues). You'll use the enhancement proposals repo if you're interested in substantially changing the way conda-forge operates.
+
+The main chat rooms that you'll interact with are:
```
Comment (beckermr) [inline src/orga/getting-in-touch.rst:11]: bot gitter room?

File: src/orga/getting-in-touch.rst [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+* [gitter: general](https://gitter.im/conda-forge/conda-forge.github.io). Our general chat room for all things conda-forge. Pretty much any question can be asked here and others in the community may be able to help.
+  Move stuff to an issue tracker if your question isn't getting resolved.
+* [gitter: core/private](https://gitter.im/conda-forge/core). Private chat for the @conda-forge/core team to discuss continued operations and improvements to conda-forge.
+* [gitter: power pc](https://gitter.im/conda-forge-ppc64le/Lobby). Public chat room for all things related to building for power pc (IBM) systems.
+
+Mailing lists:
+* [google group: conda-forge](https://groups.google.com/g/conda-forge). The general mailing list for conda-forge.
+  Sees a moderate amount of traffic, though substantially lower than the public gitter room.
+* [google group: conda-forge-core](https://groups.google.com/g/conda-forge-core). The mailing list for the core team.
```
Comment (beckermr) [inline src/orga/getting-in-touch.rst:20]: I think we want this private

File: src/orga/getting-in-touch.rst [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  Send announcements here (like meetings / CFEP stuff / votes, etc.)
+
+Other communication methods:
```
Comment (beckermr) [inline src/orga/getting-in-touch.rst:23]: not needed imho

File: index.html [hunk 4]
```diff
@@ -77,6 +77,9 @@
                     <li>
                         <a href="status">Status</a>
                     </li>
+                    <li>
+                        <a href="docs/orga/getting-in-touch.html">Contact</a>
+                    </li>
```
Comment (beckermr) [inline index.html:82]: this header needs to be added to the other html pages

File: src/orga/getting-in-touch.rst (hunk 3 above)
Comment (mariusvniekerk) [reply src/orga/getting-in-touch.rst:23]: But not particularly hamful

Comment (beckermr) [reply src/orga/getting-in-touch.rst:23]: agreed

File: src/orga/getting-in-touch.rst (hunk 1 above)
Comment (ericdill) [reply src/orga/getting-in-touch.rst:11]: Can you tell me the room name?

Comment (beckermr) [reply src/orga/getting-in-touch.rst:11]: yup. 

conda-forge/regro-cf-autotick-bot

File: index.html (hunk 4 above)
Comment (ericdill) [reply index.html:82]: only places I found with this header was
* `feedstocks.html.tmpl`
* `index.html`
* `404.html`

Did I miss any?

Comment (beckermr) [reply index.html:82]: I think that is it. The rest of the pages are external links.

**Comment by ericdill** [comment]
Closes #1146

**Pull request merged**